coldfront/core/utils/management/commands/add_scheduled_tasks.py

```diff
@@ -16,10 +16,24 @@ def handle(self, *args, **options):
 
         date = timezone.now() + datetime.timedelta(days=1)
         date = date.replace(hour=0, minute=0, second=0, microsecond=0)
-        schedule('coldfront.core.allocation.tasks.update_statuses',
-                 schedule_type=Schedule.DAILY,
-                 next_run=date)
+        schedule(
+            "coldfront.core.allocation.tasks.update_statuses",
+            schedule_type=Schedule.DAILY,
+            next_run=date,
+        )
 
-        schedule('coldfront.core.allocation.tasks.send_expiry_emails',
-                 schedule_type=Schedule.DAILY,
-                 next_run=date)
+        schedule(
+            "coldfront.core.allocation.tasks.send_expiry_emails",
+            schedule_type=Schedule.DAILY,
+            next_run=date,
+        )
+
+        schedule(
+            "coldfront.core.group.tasks.grant_usersupport_global_project_manager",
+            name="Grant RIS-UserSupport Manager status over all Projects",
+            schedule_type=Schedule.MINUTES,
+            minutes=5,
+            next_run=(timezone.now() + datetime.timedelta(minutes=5)).replace(
```

Comment:
@gunnarwustl is it necessary to indicate the next run since the task should be executed every 5 minutes?

Reply:
It seems to have been done in all prior examples and when creating a task in the UI you cannot set a NULL value.